mama_ldscores.py

```diff
@@ -11,4 +11,3 @@
 
 if __name__ == '__main__':
     main_func(sys.argv)
```

Comment:
New line below?

Reply:
I think there is one there, right?  (I don't see a Github complaint that there isn't)  There were two newlines before, so I deleted one, but did intend to leave one.